app/client/desktop/src/main/java/com/ttbmp/cinehub/app/client/desktop/ui/buyticket/choosemovie/ChooseMovieViewController.java

```diff
@@ -0,0 +1,111 @@
+import javafx.scene.control.Button;
+import javafx.scene.control.DatePicker;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListView;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.VBox;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+
+/**
+ * @author Palmieri Ivan
+ */
+public class ChooseMovieViewController extends ViewController {
+
+    private BuyTicketViewModel viewModel;
+
+
+    @FXML
+    private VBox appBar;
+
+    @FXML
+    private AppBarViewController appBarController;
+
+    @FXML
+    private ImageView theMovieDbLogoImageView;
+    @FXML
+    private Button todayButton;
+
+    @FXML
+    private Button previousButton;
+
+    @FXML
+    private Button nextButton;
+    @FXML
+    private Label errorSectionLabel;
+    @FXML
+    private ListView<MovieDto> movieListView;
+    @FXML
+    private Button confirmMovieButton;
+    @FXML
+    private DatePicker dateOfProjectionDatePicker;
+
+
+    @Override
+    protected void onLoad() {
+        appBarController.load(activity, navController);
+        viewModel = activity.getViewModel(BuyTicketViewModel.class);
+        activity.getUseCase(BuyTicketUseCase.class).getListMovie(viewModel.selectedDateProperty().getValue().toString());
+        movieListView.itemsProperty().addListener(l -> movieListView.refresh());
+        movieListView.setItems(viewModel.getMovieList());
+        movieListView.setCellFactory(movieList -> new ChooseMovieListCell(activity, navController));
+        bind();
+        dateOfProjectionDatePicker.setValue(LocalDate.now());
+        dateOfProjectionDatePicker.setDayCellFactory(CustomDateCell::new);
+        dateOfProjectionDatePicker.setOnAction(l->onDataChange());
+        todayButton.setOnAction(a -> dateOfProjectionDatePicker.setValue(LocalDate.now()));
+        previousButton.setOnAction(a -> dateOfProjectionDatePicker.setValue(dateOfProjectionDatePicker.getValue().minusDays(1)));
+        nextButton.setOnAction(a ->dateOfProjectionDatePicker.setValue(dateOfProjectionDatePicker.getValue().plusDays(1)));
+        previousButton.setDisable(true);
+        confirmMovieButton.setOnAction(a -> {
+            try {
+                activity.getUseCase(BuyTicketUseCase.class).getListCinema(
+                        new GetListCinemaRequest(
+                                viewModel.selectedMovieProperty().getValue(),
+                                viewModel.selectedDateProperty().getValue().toString()
+                        )
+                );
+                navController.navigate(new NavDestination(new ChooseCinemaView()));
+
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        });
+
+
+
+    }
+
+    private void onDataChange() {
+        activity.getUseCase(BuyTicketUseCase.class).getListMovie(viewModel.selectedDateProperty().getValue().toString());
+        previousButton.setDisable(dateOfProjectionDatePicker.getValue().equals(LocalDate.now()));
```

Comment:
The disabled property should be bonded with a ViewModel attribute to allow unit testing.